lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+Be sure to always label arrows, as this is a standard convention and improves readability.
+
+}
+
+So far, our diagram offers two unique pathways from the top of the tree to two unique items.
+
+Put another way: __There are two branches that split from the tree's root. Each branch leads to a decision node, which eventually leads to a terminal node (or "leaf").__
+
+@lesson-instruction{
+Let's identify the root, branches, nodes, and leaves on our tree so far.
+}
+
+@teacher{Discuss your in-progress tree to help students locate the different parts. The parts are intuitively named, but formal definitions are below if needed.}
+
+@right{@image{images/terminology-tree.png, 300}}
+
+- The root node is the very top node that represents the entire population or sample.
+- Splitting is the process of dividing a node into sub-nodes with branches.
+- Sub-nodes (or decision nodes) split from the root node, or from other nodes.
+- A leaf node (or terminal node) is a node that does not split.
+
+@vspace{1ex}
+
+@lesson-instruction{
+- With a partner, complete the decision tree on a piece of paper so that all six items are categorized.
+- Now, draw an entirely different decision tree - one which has a __different__ question at its root.
+}
+
+@teacher{
+
+@right{@image{images/tree3.png, 400}}
+
+One possible completed tree is on the right.
+
+To create a different tree, students can start with any of the questions from levels 2 or 3, or they can generate an entirely different starting question.
+
+As students finish, invite them to draw their trees on the board. Ideally, you will have a wide assortment of trees! If students are not developing interesting trees, urge them to think of *entirely* different questions than those posed in the sample tree.
+
+}
+
+@QandA{
+@Q{After looking at the decision trees of your classmates: What do these trees all have in common? How are they different?}
+@A{Answers will vary. Each tree will have twice as many branches as nodes. Many trees will have the same number of nodes, although probably not all. Many questions asked will likely be the same, but not every question.}
+@Q{Are all of the trees equally efficient?}
+@A{The trees are probably similarly efficient, requiring either five or six nodes, including the root. Five nodes is more efficient than six.}
+}
+
+You might be wondering: Would it be possible to make this tree any more efficient? If so, why not?!
+
+@lesson-instruction{
+- With a partner, complete @printable-exercise{comparing-trees.adoc}.
+}
+
+As we increase a tree's efficiency, we can lose accuracy. Conversely, if we focus *too* much on accuracy - making our decision trees bigger and more complex - we risk __overfitting__. Overfitting happens when we teach a model the specific quirks of one particular dataset, preventing it from making reliable predictions about new data.
```

Comment:
Something is missing here, which makes it hard to understand the relationship between accuracy and overfitting. It could be that accuracy really isn't defined, but it could also be that there isn't a good enough example of a tree that's over-fit.

Reply:
Original hope was to add a section on overfitting.
I'm concerned that adding an example of overfitting right here would make this lesson too long / intense. (It's already got a lot going on.)